navigation/create_control/scripts/ros_control_republishers.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/python
```

Comment:
From [this issue](https://github.com/ros-controls/ros_controllers/issues/126), you can use this:

http://wiki.ros.org/topic_tools/relay

Reply:
Also, keep in mind that topic names shouldn't be hardcoded.

Reply:
You should add this package to `packages.txt`.

Reply:
Done